cmd/minishift/cmd/start_test.go

```diff
@@ -116,57 +128,62 @@ func (r *RecordingRunner) Output(command string, args ...string) ([]byte, error)
 	return ocClusterUpHelp, nil
 }
 
-var testDir string
-var testRunner *RecordingRunner
-var testMachineConfig = cluster.MachineConfig{
-	OpenShiftVersion: "v1.3.1",
-}
-var testIp = "192.168.99.42"
+var (
+	testConfig = &clusterup.ClusterUpConfig{
+		OpenShiftVersion: "v1.5.1",
+		Ip:               "192.168.99.42",
+		OcPath:           "/home/john/.minishift/cache/oc/1.5.0/oc",
```

Comment:
@hferentschik , if that is the case, let's put dummy string here then. This path make me feel like we might be coming some version here :)